src/api/login.js

```diff
@@ -0,0 +1,78 @@
+export function saveUser(uUID, uName, keep) {
+  const ttl = keep ? 336 : 2;
+  const expiryDate = new Date();
+  expiryDate.setHours(expiryDate.getHours() + ttl);
+
+  const item = {
+    userUID: uUID,
+    userName: uName,
+    keepLogin: keep,
+  };
+
+  localStorage.setItem('curLogin', JSON.stringify(item));
+  localStorage.setItem('loginExpiry', expiryDate.getTime());
+}
+
+export function deleteUser() {
+  if (localStorage.getItem('curLogin')) localStorage.removeItem('curLogin');
+  if (localStorage.getItem('loginExpiry')) localStorage.removeItem('loginExpiry');
+}
+
+export function checkIfUserExists() {
+  if (localStorage.getItem('curLogin')) return true;
+  else return false;
+}
+
+export function checkUserTime() {
+  if (localStorage.getItem('curLogin') && localStorage.getItem('loginExpiry')) {
+    const expiry = localStorage.getItem('loginExpiry');
+    const now = new Date();
+    if (now.getTime() > expiry) deleteUser();
```

Comment:
explique a logica por favor

Reply:
Caso existam os itens curLogin e loginExpiry
Cria-se uma constante pra armazenar o valor de loginExpiry
Cria-se uma constante pra armazenar a hora em que o site foi aberto
Caso o valor da hora atual for maior que o valor de "expiry", deleta-se o usuário